src/Orleans.PlatformServices/Orleans.PlatformServices.csproj

```diff
@@ -21,10 +25,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.Runtime" Version="4.3.0" />
+    <PackageReference Include="System.Runtime" Version="$(SystemRuntimeVersion)" />
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.5'">
-    <PackageReference Include="System.Runtime.Loader" Version="4.3.0" />
+    <PackageReference Include="System.Runtime.Loader" Version="$(SystemRuntimeVersion)" />
```

Comment:
It is intentional we reference every System* package is using the ```SystemRuntimeVersion``` variable. ```SystemVersion``` did not make sense, hence the name. Here it is "unfortunate" that the package starts with ```System.Runtime```.